### PR TITLE
UpdateAuthorCommand and Handler added

### LINKED ISSUE #10: Commands : Create `UpdateAuthorCommand` with `UpdateAuthorCommandHandler`

### PR DESCRIPTION
**Pull request by Asadullokh27**
Closed #10

### CODE REVIEW

**Comment by Asadullokh27** [comment]
Mission Completed

File: src/BookStore.Application/useCases/Books/Commands/UpdateAuthorCommandHandler.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookStore.Application.useCases.Books.Commands
+{
+    public class UpdateAuthorCommandHandler: IRequestHandler<UpdateAuthorCommand, Author>
+    {
+
+        private readonly IAppDbContext _context;
+        private readonly IMapper _mapper;
```
Comment (QosimovIbrohim) [inline src/BookStore.Application/useCases/Books/Commands/UpdateAuthorCommandHandler.cs:18]: Mapster qayerga qarayapti

File: src/BookStore.Application/useCases/Books/Commands/UpdateAuthorCommandHandler.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            if (author is not null)
+            {
+                author.Name = request.Name;
+                author.Description = request.Description;
+
+                await _context.SaveChangesAsync();
+            }
+
+            return null;
+        }
```
Comment (QosimovIbrohim) [inline src/BookStore.Application/useCases/Books/Commands/UpdateAuthorCommandHandler.cs:38]: https://upper.uz/web/articles/45

Comment (QosimovIbrohim) [reply src/BookStore.Application/useCases/Books/Commands/UpdateAuthorCommandHandler.cs:34]: Nega return null- "ehh"

**Review by SardorSohinazarov — APPROVED** [review]

**Pull request merged**